src/app/trips/app-destinations-list/app-destinations-list.component.html

```diff
@@ -0,0 +1,19 @@
+<ng-container *ngFor="let destination of destinations">
+
+  <div [ngClass]="{'frame' : showFrame}">
```

Comment:
a more popular style for class binding would be:
``` html
<div [class.frame]="showFrame">
...
</div>
```

https://angular.io/guide/class-binding